.github/workflows/changed_solutions.yml

```
@@ -0,0 +1,61 @@
+name: Changed Solutions
```

Nit: a little more clear as a name, maybe? What do you think?

```suggestion
name: Test Changed Solutions
```